src/main/java/org/breedinginsight/brapi/v2/BrAPIV2Controller.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * See the NOTICE file distributed with this work for additional information regarding copyright ownership.
```

Comment:
Can you put a line break in after information for these new headers?

Reply:
Changed and pushed